tests/subsys/fs/fs_api/src/test_multi_fs.c

```diff
@@ -105,12 +115,39 @@ static int test_fs_external(void)
 /**
  * @brief Multi file systems register and unregister
  *
- * @details register and unregister two file systems to test
- *          the system support multiple file system simultanously
+ * @details
+ * Test Objective:
+ * - register and unregister two file systems to test the system support
+ *   multiple file system simultanously
+ *
+ * Testing techniques:
```

Comment:
You probably want to capitalize the "techniques", as you do with all words of titles in all other sections.